### DISCUSSION TITLE
Ability to remove smart filters from the customize menu

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#3349] [Ideas]
### Idea Description

We have a lot of options when it comes to adding smart filters to various parts of the UI, be the dashboard or the sidenav. What is currently lacking is the ability to remove the filters from these spots when we are done with them. 

For example here in the customize - sidenav section:

![image](https://github.com/user-attachments/assets/1afe2053-bb3a-4045-9d7b-590cdc53dce2)

I can hide the smart list, but I can't get rid of it anymore. What I'm asking for is a way to remove these lists that might of been added as a 1 off thing. They could get sent back down to the dedicated "Smart Filters" area once removed. 

### Idea Category

Feature Enhancement

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
![image](https://github.com/user-attachments/assets/72039bee-5a0c-4956-8308-85cfc4ae4e70)

This will be released with v0.8.7
